src/test/java/com/hillel/items_exchange/controller/UserControllerTest.java

```diff
@@ -234,6 +262,28 @@ void addChildren_Success_ShouldReturnHttpStatusOk() throws Exception {
                 .andExpect(jsonPath("$[1].sex").value("FEMALE"));
     }
 
+    @Test
+    @WithMockUser(username = "deletedUser")
+    @Transactional
+    @DataSet(value = {"database_init.yml", "user/deleted_user_init.yml"})
+    void addChildren_WhenUserHasStatusDeleted_ShouldReturn403WithSpecificMessage() throws Exception {
+        MvcResult mvcResult = mockMvc.perform(post("/user/child/")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(validCreatingChildDtoJson)
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isForbidden())
+                .andReturn();
+
+        User deletedUser = userService.findByUsernameOrEmail("deletedUser@gmail.com").orElseThrow();
+        String contentAsString = mvcResult.getResponse().getContentAsString();
+
+        assertTrue(contentAsString.contains(getMessageSource("exception.illegal.operation")
+                .concat(". ")
+                .concat(getParametrizedMessageSource("account.deleted.first",
+                        userService.getDaysBeforeDeletion(deletedUser)))));
+    }
```

Comment:
Code duplicate

Reply:
I'm sorry, but I did not understand anything...